<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
    <item>
     <layout class="QVBoxLayout" name="left_tab" stretch="0,0,10">
      <item>
       <widget class="QGroupBox" name="sensorBox">
        <property name="title">
         <string>Seleção do sensor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="dados_sensor">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QPushButton" name="sensorSelected">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Selecionar modelo ...</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Ângulo de partida</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_angle_start">
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="text">
                <string>N/A</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Ângulo de chegada</string>
               </property>
             </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_angle_end">
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="tabletTracking">
                <bool>false</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="text">
                <string>N/A</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Sentido de giro</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_turn_direction">
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="tabletTracking">
                <bool>false</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="text">
                <string>N/A</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="dial_control" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
         <item>
          <widget class="QFrame" name="dial_elements">
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0">
                <item>
                 <widget class="QFrame" name="frame">
                  <property name="frameShape">
                   <enum>QFrame::Shape::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_8">
                       <item>
                        <widget class="QPushButton" name="animate_dial_button">
                         <property name="text">
                          <string>Animar</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QProgressBar" name="animate_progress_bar">
                       <property name="value">
                        <number>0</number>
                       </property>
                       <property name="invertedAppearance">
                        <bool>false</bool>
                       </property>
                       <property name="textDirection">
                        <enum>QProgressBar::Direction::TopToBottom</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <property name="rightMargin">
                     <number>10</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Angulo desejado:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="degreeInsertDial"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <property name="rightMargin">
                     <number>10</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Ângulo atual:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="actual_dial_degree">
                      <property name="cursor">
                       <cursorShape>PointingHandCursor</cursorShape>
                      </property>
                      <property name="text">
                       <string>N/A</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QDial" name="plan_dial">
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="notchTarget">
               <double>3.700000000000000</double>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="right_tab" stretch="0">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Servo Minas</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="enable_servo_communication">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Habilitar comandos ao servo</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QPushButton" name="init_servo_button">
                   <property name="text">
                    <string>Inicializar Servo Minas</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="disable_servo_button">
                   <property name="text">
                    <string>Desabilitar Servo Minas</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="homing_button">
                   <property name="text">
                    <string>homing</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="clear_errors">
                   <property name="text">
                    <string>Limpar erros</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QWidget" name="logPlaceHolder" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QTabWidget" name="log_tab">
                    <property name="currentIndex">
                     <number>1</number>
                    </property>
                    <widget class="QWidget" name="servo_output">
                     <attribute name="title">
                      <string>Servo log</string>
                     </attribute>
                     <layout class="QVBoxLayout" name="verticalLayout_10">
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_8">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_11">
                          <item>
                           <widget class="QLineEdit" name="search_line_log">
                            <property name="sizePolicy">
                             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                              <horstretch>0</horstretch>
                              <verstretch>0</verstretch>
                             </sizepolicy>
                            </property>
                            <property name="inputMask">
                             <string/>
                            </property>
                            <property name="text">
                             <string extracomment="wadwd">Buscar no log do servo...</string>
                            </property>
                            <property name="echoMode">
                             <enum>QLineEdit::EchoMode::Normal</enum>
                            </property>
                            <property name="cursorPosition">
                             <number>25</number>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QPushButton" name="search_log_button">
                            <property name="text">
                             <string>Buscar</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_10">
                          <item>
                           <widget class="QTextEdit" name="log_screen"/>
                          </item>
                          <item>
                           <widget class="QScrollBar" name="log_scroll_bar">
                            <property name="orientation">
                             <enum>Qt::Orientation::Vertical</enum>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item alignment="Qt::AlignmentFlag::AlignLeft">
                         <widget class="QPushButton" name="clean_log_button">
                          <property name="text">
                           <string>Limpar log</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="general_output">
                     <attribute name="title">
                      <string>Saídas gerais</string>
                     </attribute>
                     <layout class="QVBoxLayout" name="verticalLayout_14">
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_13">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_12">
                          <item>
                           <widget class="QLineEdit" name="general_log_text_edit">
                            <property name="text">
                             <string>Buscar nas saídas...</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QPushButton" name="general_log_search_button">
                            <property name="text">
                             <string>Buscar</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_14">
                          <item>
                           <widget class="QTextEdit" name="general_log_screen"/>
                          </item>
                          <item>
                           <widget class="QScrollBar" name="verticalScrollBar_2">
                            <property name="orientation">
                             <enum>Qt::Orientation::Vertical</enum>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item alignment="Qt::AlignmentFlag::AlignRight">
                         <widget class="QPushButton" name="clean_all_output">
                          <property name="text">
                           <string>Limpar saída</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <property name="enabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QMenu" name="menuSensor">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Sensor</string>
    </property>
    <property name="enabled">
     <bool>true</bool>
    </property>
    <addaction name="actionVer_sensores"/>
    <addaction name="actionAdicionar"/>
    <addaction name="actionRemover"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSensor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVer_sensores">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
   <property name="text">
    <string>Ver sensores</string>
   </property>
  </action>
  <action name="actionAdicionar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="actionRemover">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>Remover</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
